docs/source/algorithms/LoadInstrument-v1.rst

```diff
@@ -9,51 +9,82 @@
+the given date the file with the most recent from date is selected. If no such
+files are found the file with the latest from date is selected.
+
+If only the ``InstrumentName`` is specified, a filename is built on-the-fly to
+have the form ``InstrumentName + _Definition.(xml|hdf5|nxs)``. Short instrument
+names can also be used, such as ``seq``; they get translated to long instrument
+names (e.g. ``SEQUOIA``) through the ``ConfigServiceImp::getInstrument().name()``
+method.
 
 Usage
 -----
 
 **Example - Load instrument to a workspace:**
 
 .. testcode:: exLoadInstrument
-   
-   # create sample workspace
-   ws=CreateSampleWorkspace();
-   inst0=ws.getInstrument();
-
-   print("Default workspace has instrument: {0} with {1} parameters".format(inst0.getName(),len(inst0.getParameterNames())))
-
-   # load MARI
-   det=LoadInstrument(ws,InstrumentName='MARI', RewriteSpectraMap=True)
-   inst1=ws.getInstrument();
-
-   print("Modified workspace has instrument: {0} with {1} parameters".format(inst1.getName(),len(inst1.getParameterNames())))
-   print("Instrument {0} has the following detectors:  {1}".format(inst1.getName(), det))
 
+   # create sample workspace
+   ws1 = CreateSampleWorkspace();
+   inst1 = ws1.getInstrument();
+   print("Default workspace has instrument: {0} with {1} parameters".format(inst1.getName(),len(inst1.getParameterNames())))
+
+   # load MARI from instrument name
+   print("===========================")
+   mon1 = LoadInstrument(ws1, InstrumentName="MARI", RewriteSpectraMap=True)
+   inst1 = ws1.getInstrument()
+   di1 = ws1.detectorInfo()
+   ci1 = ws1.componentInfo()
+   print("Modified workspace {0} has instrument: {1}".format(ws1.getName(), inst1.getName()))
+   print("Instrument {0} has {1} components, including {2} monitors and {3} detectors".format(inst1.getName(), ci1.size(), len(mon1), di1.size()))
+
+   # load LOKI from file name
+   print("===========================")
+   ws2 = CreateSampleWorkspace();
+   mon2 = LoadInstrument(ws2, FileName="LOKI_Definition.hdf5", RewriteSpectraMap=True)
+   inst2 = ws2.getInstrument()
+   di2 = ws2.detectorInfo()
+   ci2 = ws2.componentInfo()
+   print("Workspace {0} has instrument: {1}".format(ws2.getName(), inst2.getName()))
+   print("Instrument {0} has {1} components, including {2} monitors and {3} detectors".format(inst2.getName(), ci2.size(), len(mon2), di2.size()))
 
 **Output:**
 
 .. testoutput:: exLoadInstrument
 
    Default workspace has instrument: basic_rect with 0 parameters
-   Modified workspace has instrument: MARI with 74 parameters
-   Instrument MARI has the following detectors:  [1 2 3]
+   ===========================
+   Modified workspace ws1 has instrument: MARI
+   Instrument MARI has 963 components, including 3 monitors and 921 detectors
+   ===========================
+   Workspace ws2 has instrument: LOKI
+   Instrument LOKI has 8011 components, including 0 monitors and 8000 detectors
```

Comment:
Well done on the documentation for this. Mainly very good. Really nice to see the effort put in here.